api/models.py

```diff
@@ -78,14 +77,17 @@ class User(db.Model):
     twitter_username_verified = db.Column(db.Boolean, nullable=False, default=False)
     twitter_username_verification_tweet_id = db.Column(db.String(64), nullable=True)
 
+    store_name = db.Column(db.String(100), unique=True, nullable=True, index=True)
+    store_description = db.Column(db.String(240), unique=True, nullable=True)
```

Comment:
No unique on any of these.

Reply:
deleting these based on discussion in telegram